DuckDB.NET/DuckDB.NET.csproj

```diff
@@ -43,12 +45,27 @@
       <PackagePath></PackagePath>
     </None>
   </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
   </ItemGroup>
-
+  
+  <!-- Download and include the native libraries into the nuget package-->
+  <Target Name="DownloadNativeLibs" BeforeTargets="GenerateAdditionalSources">
+    <MSBuild Projects="DownloadNativeLibs.targets" Properties="Rid=win-x86;LibUrl=https://github.com/duckdb/duckdb/releases/download/v0.6.1/libduckdb-windows-i386.zip" />
```

Comment:
Instead of hard-coding a specific version, is it possible to download the latest release?

Reply:
Just checked. Doesn't seem to be possible.
Seems they don't tag the latest release.